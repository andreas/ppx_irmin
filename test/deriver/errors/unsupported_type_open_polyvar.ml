type 'a t = [> `Alpha | `Beta ] as 'a [@@deriving irmin]
