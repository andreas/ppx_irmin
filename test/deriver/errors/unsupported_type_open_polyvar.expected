File "unsupported_type_open_polyvar.ml", line 1, characters 0-56:
Error: ppx_irmin: type alias "[> `Alpha  | `Beta ] as 'a" is not supported.
